### ISSUE TITLE
example server usage?

### ISSUE THREAD

**Issue by timothydmorton** [ioam/parambokeh#1]
Can you put up a minimal working example (e.g., in a doc notebook) of how to use parambokeh with the bokeh server?  I've been trying it out in the notebook but I can't get it to work with a server.

**Comment by ceball** [comment]
Yes, we will do this once we've updated the code to work with bokeh 0.12.10.
